javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,120 @@
+/* My age in 2027*/ 
+let yearOfBerth;
+let yearFuture;
+yearOfBerth = 1985; // integer, 
+yearFuture = 2027; // integer, 
+const age = yearFuture - yearOfBerth;
+console.log('You will be' + age+ 'years old in' + yearFuture);
```

Comment:
A nice thing to know is that there is an alternative to using + to concatenate strings, it's called template strings: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals 
In short, you could replace it like this:
```
console.log(`You will be ${age} years old in ${yearFuture}`)
```